frameworks/Rust/thruster/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "thruster_techempower"
+version = "0.1.0"
+authors = ["Isaac Whitfield <iw@whitfin.io>"]
+
+[dependencies]
+futures = "0.1"
+serde = "1.0"
+serde_derive = "1.0"
+serde_json = "1.0"
+thruster = { git = "https://github.com/trezm/Thruster" }
```

Comment:
Let's lock this down to a version -- 4.4 for now, just so that when master receives a breaking change we don't break these benchmarks :)

Reply:
@trezm I'm going to update this but for the time being only master supports compilation on stable - I'll pin to 0.4.5 once released.

Reply:
Just release 0.4.5 if you want to bump this